src/api/common/schemas/ledger-options.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "ledger-options",
+  "description": "Options for getLedger",
+  "type": "object",
+  "properties": {
+    "ledgerVersion": {"$ref": "ledgerVersion"},
+    "includeAllData": {"type": "boolean"},
```

Comment:
4am here, just woke up from weird dreams, drawing blanks, nothing I like popping out

You could flip it out around to saying something about including only ids/hashes,  e.g. itemIdentifiersOnly which defaults to true. 

I got nothing haha